Source/Core/Duality/Drawing/Canvas.cs

```diff
@@ -1026,7 +1026,8 @@ public void FillRect(float x, float y, float width, float height)
 		/// <param name="blockAlign">Specifies the alignment of the text block.</param>
 		public void DrawText(string text, float x, float y, float z = 0.0f, Alignment blockAlign = Alignment.TopLeft, bool drawBackground = false)
 		{
-			this.DrawText(new string[] { text }, x, y, z, blockAlign, drawBackground);
+			if(!string.IsNullOrEmpty(text))
```

Comment:
Next time, please add a space between the if and the opening bracket! :) Merging now anyway.